librz/include/rz_util/rz_ebcdic.h

```diff
@@ -0,0 +1,469 @@
+// SPDX-FileCopyrightText: 2021 Billow <billow.fun@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RZ_EBCDIC_H
+#define RZ_EBCDIC_H
+
+#include "rz_utf8.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static const RzRune ibm037_to_uni[256] = {
```

Comment:
I think it's better to move this data to the C file instead, to avoid multiple inclusion, etc. Users of the `rz_util` headers don't need these tables.